packages/jest-snapshot/src/index.js

```diff
@@ -92,7 +92,9 @@ const toMatchSnapshot = function(received: any, testName?: string) {
 
     report = () =>
       `${RECEIVED_COLOR('Received value')} does not match ` +
-      `${EXPECTED_COLOR('stored snapshot ' + count)}.\n\n` +
+      `${EXPECTED_COLOR(
+        `stored snapshot "${testName ? testName + ' ' : ''}${count}"`,
```

Comment:
Looking at the failing CI, I think it looks better if we only wrap in quotes if there's a `testName`. Thoughts?

So it should be:

```js
   `${EXPECTED_COLOR(
        `stored snapshot ${testName ? `"${testName} ${count}"` : count}`,
      )}.\n\n` +
```